server/tests/audit_math/test_minerva.py

```diff
@@ -258,7 +258,7 @@ def test_compute_risk_close_narrow():
         minerva.make_sample_results(c, [[5100, 4990, 1]]),
         {1: RoundInfo("r0", 10091)},
     )
-    assert res == ({("winner", "loser"): 0.16896200607848647}, False)
+    assert res == ({("winner", "loser"): 0.1689620060785905}, False)
```

Comment:
Annoyingly, this is now failing in CI:

```
>       assert res == ({("winner", "loser"): 0.1689620060785905}, False)
E       AssertionError: assert ({('winner', 'loser'): 0.16896200607859224}, False) == ({('winner', 'loser'): 0.1689620060785905}, False)
E         At index 0 diff: {('winner', 'loser'): 0.16896200607859224} != {('winner', 'loser'): 0.1689620060785905}
E         Full diff:
E         - ({('winner', 'loser'): 0.1689620060785905}, False)
E         ?                                        ^^
E         + ({('winner', 'loser'): 0.16896200607859224}, False)
E         ?                                        ^^^
```

It seems to have a _different_ value than either the old one or the new one I saw locally.